### PR TITLE
Add "tests" install group, add empty init file

### PR DESCRIPTION
**Pull request by kinow**
Related to #2 

With these changes `python3 -m pytest -v` command completes successfully on my environment :+1:

```bash
python -m venv venv
source venv/bin/activate
pip install -e .[all]
python3 -m pytest -v
```

Alternatively, it's also possible to run `pip install -e .[tests]`, and perhaps we could add a dependency group for docs too, with `sphinx` and friends?

Cheers
-Bruno

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -24,6 +24,12 @@ package_dir =
 tests_require =
     pytest
 
+[options.extras_require]
+tests =
+    pytest
+all =
+    %(tests)s
+
```
Comment (kinow) [inline setup.cfg:32]: Based on https://github.com/ecmwf/pyflow/blob/d3585c393e6bba6b9ba544f8b38c40447245488d/setup.cfg#L23-L30
